### ISSUE TITLE
Support cordova-plugin-add-swift-support@2.0.2

### ISSUE THREAD

**Issue by saxxi** [pushy-me/pushy-cordova#18]
I've manually updated `cordova-plugin-add-swift-support` to latest version (`2.0.2` at present time) in order to avoid the following error:

```
Using “requireCordovaModule” to load non-cordova module “glob” is not supported
```

However now running `cordova plugin add pushy-cordova` will report that `1.7.2` is required as per [plugin.xml](https://github.com/pushy-me/pushy-cordova/blob/master/plugin.xml) version lock.

Is the upgrade possible?
Thank you.

**Comment by pushy-me** [comment]
Hi @saxxi,
Thanks so much for your report.

We've just updated the `cordova-plugin-add-swift-support` to version `2.0.2` as per your recommendation and published version `1.0.27` to npm.

Please update as follows:
```
cordova plugin add pushy-cordova@1.0.24
```

Then, rebuild your project and check if the issue is resolved.

**Comment by pushy** [comment]
Hi @saxxi,
Thanks so much for your report.

We've just updated the `cordova-plugin-add-swift-support` to version `2.0.2` in https://github.com/pushy-me/pushy-cordova/commit/3e72d62656f34aec694d3c16085b2dc795a69de8 as per your recommendation and published version `1.0.27` to npm.

Please update as follows:
```
cordova plugin add pushy-cordova@1.0.27
```

Then, rebuild your project and check if the issue is resolved.
